x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_chart_flyout/datafeed_chart_flyout.tsx

```diff
@@ -595,3 +597,85 @@ export const DatafeedChartFlyout: FC<DatafeedChartFlyoutProps> = ({
     </EuiPortal>
   );
 };
+
+interface JobListDatafeedChartFlyoutProps {
+  setShowFunction: (callback: Function) => void;
```

Comment:
the type for `callback` should probably be the same as the other flyouts on the jobs list:
https://github.com/peteharverson/kibana/blob/6a16228838d4be5db7cb466a1ce6e5a3d29cd6bc/x-pack/plugins/ml/public/application/jobs/jobs_list/components/confirm_modals/close_jobs_confirm_modal.tsx#L29

Reply:
Done in https://github.com/elastic/kibana/pull/136274/commits/b458fea673c51b18a247d5985d894f9d629d6b62